### PR TITLE
Better Beacon and Instance lists

### PR DESCRIPTION
**Pull request by cmbrose**
### What
- Updated output of the `/beacons/list`.  Now gives aliases.
- Updated output of the `/beacons/list/{Beacon}` endpoint.  Now gives all data provided by Beacon.
- Created the `/beacons/refresh/{Beacon}` endpoint
### How

The biggest change here is a refactoring of `beacons` which split instances and beacons into 2 databases.  This helped immensely with refreshes, but also makes more sense from a model standpoint as beacons don't really need any notion of what instances they have.
### Endpoints

``` http
GET /beacons/list HTTP/1.1
```

``` http
GET /beacons/list/{Beacon} HTTP/1.1
```

**Query Parameters**
refresh - (optional boolean) polls the beacon for any new instances and repopulates the databases

``` http
PUT /beacons/refresh/{Beacon} HTTP/1.1
```
#### Examples

**Beacon listing**

Request:

``` http
GET /beacons/list HTTP/1.1
```

Response:

``` http
HTTP/1.1 200 OK
[ 
    { 
        "Alias" : "LocalBeacon",
        "Address": "127.0.0.1:5002"
    }
]
```

**Instance listing**

Request:

``` http
GET /beacons/list/LocalBeacon HTTP/1.1
```

Response:

``` http
HTTP/1.1 200 OK
[
    {
        "InstanceAddress": "127.0.0.1:5001/v1.12", 
        "Name": "boot2docker", 
        "CanAccessDocker": true, 
        "BeaconAddress": "127.0.0.1:5002"
    }
]
```

### CODE REVIEW

**Comment by cmbrose** [comment]
@lighthouse/owners not sure why the Travis is angry, but I'll look into it later.  Anyways, for your review.

File: beacons/beacons.go [hunk 1]
```diff
@@ -73,34 +90,37 @@ func GetBeaconAddress(instance string) (string, error) {
...
-    return beacon.Token, nil
+    return data.Token, nil
 }
 
-func LoadBeacons() []beaconData {
+func LoadBeacons() {
     var fileName string
-    if _, err := os.Stat("/config/beacon_permissions.json"); os.IsNotExist(err) {
+    var err error
+    if _, err = os.Stat("./config/beacon_permissions.json.dev"); err == nil {
```
Comment (cmbrose) [inline beacons/beacons.go:121]: @ngmiller I added dev configs.  Only for beacons now, but I'll add more later on.  You're welcome :)

**Comment by cmbrose** [comment]
I should also note that this is built off https://github.com/lighthouse/lighthouse/pull/38 so the diff will go down when that goes through.

**Comment by nickmonad** [comment]
+1

**Pull request merged**